multi-glob.js

```diff
@@ -0,0 +1,40 @@
+const glob = require('glob');
+
+// Copy pasta from broccoli-kitchen-sink-helpers
+// This version doesn't throw an error on empty matches.
+// See: https://github.com/broccolijs/broccoli-kitchen-sink-helpers/pull/35
```

Comment:
By allowing this, it is possible to create a BroccoliHandlebars nodes that watch over multiple `node`s (`tree`s) that can trigger a rebuild even if the `outputFiles` have not changed.

This is useful when building static site generators like broccoli-taco. ex:

```
new BroccoliHandlebars([pages, layouts], ['default.hbs'], {
  context: function() {
    const page = Handlebars.compile(...);
    return {page};
  }
});

// default.hbs
<body>
  {{page}}
</body>
```